.github/workflows/main.yml

```diff
@@ -422,7 +423,7 @@ jobs:
         python: [3.8] # Use flaky tests to run against more versions of Python.
         # integration: Tests with VS Code, Python extension & real Jupyter
         # notebook: Notebook Tests with VS Code, Python extension & real Jupyter
-        test-suite: [integration]
+        test-suite: [integration, notebook, notebookWithoutPythonExt, conda]
```

Comment:
@rchiodo Just added this to the review. It wasn't running the vscode notebook tests in PRs anymore.